plugins/es-repository-azure/src/test/java/org/elasticsearch/repositories/azure/AzureSnapshotIntegrationTest.java

```diff
@@ -104,6 +105,48 @@ public void create_azure_snapshot_and_restore_it() {
         handler.blobs().keySet().forEach(x -> assertThat(x).doesNotEndWith("dat"));
     }
 
+    @Test
+    public void create_azure_snapshot_and_restore_with_secondary_endpoint() {
+        execute("CREATE TABLE t1 (x int)");
+        assertThat(response.rowCount()).isEqualTo(1L);
+
+        var numberOfDocs = randomLongBetween(0, 10);
+        for (int i = 0; i < numberOfDocs; i++) {
+            execute("INSERT INTO t1 (x) VALUES (?)", new Object[]{randomInt()});
+        }
```

Comment:
```suggestion
        int numberOfDocs = randomIntBetween(0, 10);
        Object[][] rows = new Object[numberOfDocs][];
        for (int i = 0; i < numberOfDocs; i++) {
            rows[i] = new Object[] { randomInt() };
        }
        execute("INSERT INTO t1 (x) VALUES (?)", rows);
```

Bulk inserts are faster

Reply:
thx!